### ISSUE TITLE
Docker image versioning

### ISSUE THREAD

**Issue by msschl** [pelias/pelias#905]
It would be nice to publish the release/tag version as docker tag next to the tag `latest`.
This would enable us to specifically pin a version for production and update manually when required.

**Comment by missinglink** [comment]
Hi @msschl we tag all branches twice, [once with the branch name](https://hub.docker.com/layers/pelias/api/master/images/sha256-b9d7ee7ba2bde983ee4d883773fc84e37e88cf928710828b6a0d72deeefd8610) (eg. `master`) and [once with a datestamp and commit hash](https://hub.docker.com/layers/pelias/api/master-2021-06-03-0f638f1323eaaf6332b105dd7087846e6926e2c3/images/sha256-b9d7ee7ba2bde983ee4d883773fc84e37e88cf928710828b6a0d72deeefd8610). (eg. `master-2021-06-03-0f638f1323eaaf6332b105dd7087846e6926e2c3`).

That's true of all branches, including all dev branches from the repos in the `pelias` org on github.

The `latest` tag is a common docker convention, we also tag [the most current `master` branch as `latest`](https://hub.docker.com/layers/pelias/api/latest/images/sha256-b9d7ee7ba2bde983ee4d883773fc84e37e88cf928710828b6a0d72deeefd8610) for convenience. 

If you click on those three links above you'll see that they correspond to the same hash `DIGEST:sha256:b9d7ee7ba2bde983ee4d883773fc84e37e88cf928710828b6a0d72deeefd8610`.

tl;dr you can pin to `master-****`

**Comment by msschl** [comment]
Well, It looks like it would be easier for the end-user to use the version from the github tag i.e., `pelias/api:v5.26.0`.

**Comment by missinglink** [comment]
Good idea, we'd be happy to add a new tagging format in:
https://github.com/pelias/ci-tools/blob/master/build-docker-images.sh

**Comment by missinglink** [comment]
There is a command called `git tag --points-at master` but I wasn't able to get it to work 🤷

**Comment by msschl** [comment]
`git describe --abbrev=0 --tags` might also work

**Comment by missinglink** [comment]
I think we'd need to play with it a bit:

```bash
/code/pelias/api @e2cf49dd *6 ?5 ❯ git describe --abbrev=0 --tags
v5.26.0

/code/pelias/api @67103dc7 *6 ?5 ❯ git describe --abbrev=0 --tags
v5.26.0
```

It should only occur on merges to `master` and it would need to reliably detect if there was a tag which pointed to the same commit as `master@HEAD`

**Comment by orangejulius** [comment]
We got this same request a while back in https://github.com/pelias/docker/issues/212 :) It would definitely be convenient for everyone.

@msschl your change looks promising, we'll take a look, test it out, and hopefully merge it. Thanks!

**Comment by msschl** [comment]
You’re welcome 🚀
